holoviews/operation/timeseries.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import param
+
+from ..core import ElementOperation, Element
+from ..core.util import pd
```

Comment:
I think it might be better to make this explicit i.e show that ``pd`` is None if pandas isn't available.